src/optparse.coffee

```diff
@@ -18,8 +18,8 @@ exports.OptionParser = class OptionParser
   #     [short-flag, long-flag, description]
   #
   # Along with an optional banner for the usage help.
-  constructor: (rules, @banner) ->
-    @rules = buildRules rules
+  constructor: (ruleDecls, @banner) ->
```

Comment:
Yes. The SWITCHES array in command.coffee that `buildRules()` consumes isn't executable on its own, so I wanted to distinguish it from the result (a dictionary and array with long and short options).